stack/stack.c

```diff
@@ -0,0 +1,120 @@
+#include "../stack/stack.h"
+
+typedef struct StackElement
+{
+    int value;
+    struct StackElement* nextElementPtr;
+} StackElement;
+
+typedef struct Stack
+{
+    StackElement* lastElementPtr;
```

Comment:
```suggestion
    StackElement* firstElementPtr;
```
Как правило, вершину стека считают первым элементом.